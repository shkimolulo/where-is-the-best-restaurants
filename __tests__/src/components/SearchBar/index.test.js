import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

describe('SearchBar', () => {
  let SearchBar;
    
  beforeEach(() => {
      SearchBar = require('../../../../src/components/SearchBar').default;
  });

  it('렌더링 시킬 수 있다.', () => {
    // Given

    // When
    const json = renderer.create(<SearchBar />).toJSON()

    // Then
    expect(json).toMatchSnapshot();
  });
})
