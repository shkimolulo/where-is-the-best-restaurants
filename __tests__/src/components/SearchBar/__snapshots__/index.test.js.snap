// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar 렌더링 시킬 수 있다. 1`] = `
<View
  style={
    Object {
      "backgroundColor": "yellow",
      "flexDirection": "row",
      "height": 80,
      "marginTop": 100,
      "width": 300,
    }
  }
>
  <View
    style={
      Object {
        "justifyContent": "center",
        "width": 230,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      rejectResponderTermination={true}
      style={
        Object {
          "borderWidth": 2,
          "height": 50,
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "red",
        "justifyContent": "center",
        "opacity": 1,
        "width": 70,
      }
    }
  >
    <Text>
      검색
    </Text>
  </View>
</View>
`;
